matcher/solvers/minmax_solver.py

```diff
@@ -27,19 +27,27 @@ def __init__(
             maximums,
             demands,
             encoder,
+            allow_all_affinities=True,
             logger=logging.getLogger(__name__)
         ):
 
         self.minimums = minimums
         self.maximums = maximums
         self.demands = demands
         self.cost_matrix = encoder.cost_matrix
+        self.allow_all_affinities = allow_all_affinities
 
         if not self.cost_matrix.any():
             self.cost_matrix = np.random.rand(*encoder.cost_matrix.shape)
 
         self.constraint_matrix = encoder.constraint_matrix
 
+        if not self.allow_all_affinities:
```

Comment:
why do you need to this change in the MinMaxSolver?

Reply:
@ameyagodbole my understanding is that this change was needed to keep the behavior for MinMax consistent with FairFlow, but feel free to comment on this question.